nh_eobs_mobile/static/dev/less/src/button.less

```diff
@@ -180,4 +186,21 @@ Styleguide 2.5
       }
     }
   }
-}
\ No newline at end of file
+}
+
+/*
+Black Buttons With White Text
+
+.white-on-black - White text on a black background.
+*/
+
+// White on black class should be stand alone but the do-it class specifies
+// background colours and takes precedence. This is a work around until there
+// is time to refactor the do-it class to be unopinionated about colour.
```

Comment:
Replace above comment with: "Need to amend stylesheet to include a new concept for activating / deactivating patient statuses so that this visual language is reusable. From there we can have the start / stop rapid tranq buttons keep this class (as it only specifies colour) but in addition have a new class for representing patient status changes."